scene/main/scene_tree.cpp

```diff
@@ -1507,8 +1507,11 @@ void SceneTree::_live_edit_instance_node_func(const NodePath &p_parent, const St
 		Node *n2 = n->get_node(p_parent);
 
 		Node *no = ps->instance();
-		no->set_name(p_name);
+		if (!no) {
+			continue;
```

Comment:
Should maybe be `ERR_CONTINUE` too, I don't think we want to silently ignore a failure to instantiate a packed scene.

Reply:
I just copy behaviour from above https://github.com/godotengine/godot/blob/437939589234559febabb9a4c392145521a9c3a5/scene/main/scene_tree.cpp#L1474-L1480

Reply:
Alright, makes sense then I guess :)